include/casmutils/misc.hpp

```diff
@@ -63,10 +64,10 @@ template <typename BinaryComparator_f> class UnaryComparator_f
 namespace extend
 {
 /// Make a PrimClex but shut up about it
-CASM::PrimClex quiet_primclex(CASM::xtal::Structure& prim);
+// CASM::PrimClex quiet_primclex(CASM::xtal::Structure& prim);
```

Comment:
Is anything still using this? I assume no

Reply:
Mostly no. But I really haven't paid much attention to frankenstein stuff.

Reply:
If you don't intend on bringing the code back to life, you should just delete it.